Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -333,17 +335,19 @@ type PrintOptions struct {
 // received from watches.
 type HumanReadablePrinter struct {
 	handlerMap map[reflect.Type]*handlerEntry
-	options    PrintOptions
+	Options    PrintOptions
```

Comment:
None of the fields of `HumanReadablePrinter` are exported. I think exporting the `Options` and mutating it from outside is a step backwards in the design, it breaks the encapsulation.